string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,27 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  if (char === 'a' || char === 'A') {
+    return true;
+  } else if (char === 'e' || char === 'E') {
+    return true;
+  } else if (char === 'o' || char === 'O') {
+    return true;
+  } else if (char === 'u' || char === 'U') {
+    return true;
+  } else if (char === 'i' || char === 'I') {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
If you lowercase or uppercase the letter before the conditional, you can cut the amount fo items to check by half 🤓